src/scripts/migrate_asset_data.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { createConnection } = require('typeorm');
+const {
+  Asset,
+  AssetsHistory,
+} = require('../../dist/modules/assets/assets.entity.js');
+
+(async function () {
+  const connection = await createConnection({
+    // had to copy from orgmconfig because typeorm doesn't detect postgres driver
+    type: 'postgres',
```

Comment:
I would get connection parameters from the `.env` file after adding `require('dotenv').config()` at the top of the script.

Reply:
> A couple questions:
> 
> * Are there any foreign keys which refer to the assets table? Is this a safe delete from that standpoint?

I don't think there is any other dependency on Asset and AssetsHistory, I will investigate more into this.

> * How will this script be executed? Would it need to be run manually?

Yes, this needs to called manually. I will add the command in `package.json`.